multicluster/cmd/multicluster-controller/clusterset_webhook.go

```diff
@@ -57,6 +63,22 @@ func (v *clusterSetValidator) Handle(ctx context.Context, req admission.Request)
 			klog.ErrorS(err, "the field 'clusterID' is immutable", "ClusterSet", klog.KObj(clusterSet))
 			return admission.Denied("the field 'clusterID' is immutable")
 		}
+
+		if isMemberListChanged(oldClusterSet.Spec.Members, clusterSet.Spec.Members) {
+			// Don't allow general user to update member list
```

Comment:
general user -> other users

"the member list."

Reply:
Fixed